src/ros_env.hpp

```diff
@@ -89,8 +89,7 @@ static void setMasterURI( const std::string& uri, const std::string& network_int
   remap["__master"] = uri;
   remap["__ip"] = ::naoqi::ros_env::getROSIP(network_interface);
   // init ros without a sigint-handler in order to shutdown correctly by naoqi
-  const char* ns_env = std::getenv("ROS_NAMESPACE");
-  ros::init( remap, (ns_env==NULL)?(std::string("naoqi_driver_node")):(::naoqi::ros_env::getPrefix()) , ros::init_options::NoSigintHandler );
+  ros::init(remap, std::string("naoqi_driver_node"), ros::init_options::NoSigintHandler);
```

Comment:
IMO, we should still consider the namespace and the prefix option. 
The prefix option is meant to be set once the naoqi driver is loaded as a naoqi module. 
When loaded externally (as a ros module), we should still have the option of namespacing. Have a look at this one: http://wiki.ros.org/Remapping%20Arguments

Since we anyway use the remapping options, we can then also set the __ns field.
Does this make sense what I am saying?
We could